src/css/profile/mobile/common/button.less

```diff
@@ -210,8 +210,7 @@ a.ui-btn {
 			&::before {
 				opacity: 1;
 				animation: btn_press_animation_flat_opacity_in linear 100ms,
-					btn_press_animation_flat_opacity_out linear 400ms 100ms,
-					btn_press_animation_flat_scale cubic-bezier(0.33, 0, 0.2, 1) 350ms;
+					btn_press_animation_flat_opacity_out linear 400ms 100ms;
```

Comment:
I agree with @lslachciak .
It is better not to modify it unless it is functionally necessary.